### ISSUE TITLE
Define guidelines for the units

### ISSUE THREAD

**Issue by SilasK** [Snakemake-Profiles/doc#3]
Hey, thank you for this valuable resources. 

I recommend to define guidelines for the keywords to put in the rules and their units of time, memory...
Otherwise switching from one profile to the other would need to adapt all resource items in the Snakefile or do create a new snakemake-profile.

what do you think?

**Comment by johanneskoester** [comment]
I totally agree, but there will (hopefully soon) be support for resource units in Snakemake itself, see [here](https://github.com/Snakemake-Profiles/pbs-torque/issues/2#issuecomment-351718196).
